monitoring/recording-rules.yml

```
@@ -78,3 +78,83 @@ groups:
       - record: slo:projects:error_budget_remaining_pct
         expr: |
           (1 - (slo:projects:error_ratio / 0.001)) * 100
+
+# ── Business-level recording rules (issue #1144 Part B) ─────────────────────
+#
+# Pre-computes business KPIs from the business-metrics-exporter gauges so that
+# Grafana panels can reference concise recording-rule names. These rules are
+# evaluated on the same Prometheus instance and only aggregate already-aggregated
+# gauges — no individual donor data is stored in Prometheus.
+
+groups:
```

YAMLlint (1.37.1)

[error] duplication of key "groups" in mapping

(key-duplicates)